### PR TITLE
Examples for Tutorial 5 – Playing videos on tvOS

### PR DESCRIPTION
**Pull request by BohdanBilonoh**
Need to add examples for Tutorial 5 – Playing videos on tvOS from [doc](https://docs.google.com/document/d/19cVTBPadBSYWdEf6f933mdOQCrPgMU_VdG1EaTNyb2c/edit)

[JIRA Issue](https://jira.ops.aol.com/browse/OMSDK-523)

### CODE REVIEW

File: NewTutorials/Tutorial/SetupTutorials.swift [hunk 1]
```diff
@@ -6,26 +6,48 @@ import PlayerControls
 
 
 func setupPlayingVideos(tutorialCasesViewController: TutorialCasesViewController) {
+    func playSingleVideo(viewController: UIViewController) {
+        guard let wrapper = viewController as? PlayerViewControllerWrapper else { return }
```
Comment (AndriiMoskvin) [inline NewTutorials/Tutorial/SetupTutorials.swift]: Put this code at the top?

File: NewTutorials/Tutorial_tvOS/SetupTutorials.swift [hunk 2]
```diff
@@ -0,0 +1,49 @@
+//  Copyright © 2017 Oath. All rights reserved.
+
+import UIKit
+
+
+func setup(tutorialCasesViewController: TutorialCasesViewController) {
+    func playSingleVideo(viewController: UIViewController) {
+        guard let wrapper = viewController as? SystemPlayerViewControllerWrapper else { return }
```
Comment (AndriiMoskvin) [inline NewTutorials/Tutorial_tvOS/SetupTutorials.swift]: The same - put it at the top!

File: NewTutorials/Tutorial_tvOS/SystemPlayerViewControllerWrapper.swift [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+                                      handler: nil))
+                present(alert,
+                        animated: true,
+                        completion: nil)
+            }
+            
+            func render(player: Player) {
+                if let filter = props.filter {
+                    systemPlayerViewController?.contentCIFilterHandler = {
+                        guard let outputImage = filter.outputImage else { return }
```
Comment (AndriiMoskvin) [inline NewTutorials/Tutorial_tvOS/SystemPlayerViewControllerWrapper.swift]: Replace `$0` with descriptive name.

**Review by AndriiMoskvin — CHANGES_REQUESTED** [review]

**Review by AndriiMoskvin — APPROVED** [review]

**Comment by BohdanBilonoh** [comment]
Moved to https://github.com/aol-public/OneMobileSDK-playground-ios/pull/45

**Pull request closed without merging**